coordinator/cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java

```diff
@@ -278,7 +278,7 @@ private static String metric(String name) {
   private class ConnectionMetricsImpl implements ConnectionMetrics {
 
     private final Tags tags;
-    private final Counter requestsProcessed;
+    private Counter requestsProcessed;
```

Comment:
@ivansenic Connection metrics is created when new connection is created and driver info is available only when startup message is sent. So driver based tag metrics is refreshed on start up message.